packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerPlugin.java

```diff
@@ -207,6 +209,87 @@ private File scaleImage(Image image, Double maxWidth, Double maxHeight) throws I
     fileOutput.write(outputStream.toByteArray());
     fileOutput.close();
 
+    if (shouldDownscale) copyExif(image.getPath(), scaledCopyPath);
+
     return imageFile;
   }
+
+  private void copyExif(String filePathOri, String filePathDest) {
+    try {
+      ExifInterface oldexif = new ExifInterface(filePathOri);
+      ExifInterface newexif = new ExifInterface(filePathDest);
+
+      int build = Build.VERSION.SDK_INT;
+
+      // From API 11
```

Comment:
Flutter only runs on API 16+. You can skip some of the checks and use constant flags instead of strings.